### PR TITLE
Create Arch.yml

### LINKED ISSUE #189: Arch Linux support
I was able to get the role working on Arch Linux with the following variables:
```
    __postgresql_version: "13"
    __postgresql_data_dir: "/var/lib/postgres/data"
    __postgresql_bin_path: "/usr/sbin"
    __postgresql_config_path: "/var/lib/postgres/data"
    __postgresql_daemon: "postgresql"
    __postgresql_packages:
      - postgresql
```

However it does not install the postgresql package using pacman, so that has to be done manually still.

### PR DESCRIPTION
**Pull request by C0rn3j**
Fixes #189

### CODE REVIEW

**Comment by fhsctv** [comment]
@C0rn3j : I think you should extend the molecule tests in this role to test the role against an arch container.
I did something similar with Rocky Linux 9.
You may get inspiration from here: https://github.com/geerlingguy/ansible-role-postgresql/pull/224/files

**Comment by fhsctv** [comment]
@C0rn3j : No interest in fully implementing Arch Support?

**Comment by geerlingguy** [comment]
I would like to get Arch Linux supported here (and add tests to a couple other roles too), but right now I'm just marking some issues as planned that I'd _like_ to get to but can't yet. I would be glad to add on an `archlinux` testing image, though it seems Arch likes to update things quite rapidly, so maintenance (where I like to stick to a particular version, like RHEL 7/8/9) could be a bit weird.

**Comment by geerlingguy** [comment]
Alternatively, since Arch is a little weird in that regard, I'd be willing to merge this change without adding archlinux to the testing suite (for now at least... we'll see how maintenance goes long term). But please do add it in the listing for supported distros in the `meta/main.yml` file.

**Comment by C0rn3j** [comment]
I've dropped the test and added the meta.yml entry for now.

> Arch likes to update things quite rapidly, so maintenance (where I like to stick to a particular version, like RHEL 7/8/9) could be a bit weird

I think the problematic things with these minimal images would most likely be the systemd hacks, since in my Dockerfile I used pip to install Ansible itself, so it should hopefully mostly be the same as other distros.

File: meta/main.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ galaxy_info:
   license: "license (BSD, MIT)"
   min_ansible_version: 2.8
   platforms:
+    - name: Arch Linux
```
Comment (geerlingguy) [inline meta/main.yml:12]: Should be `ArchLinux`:

```suggestion
    - name: ArchLinux
```

File: meta/main.yml [hunk 2]
```diff
@@ -9,6 +9,9 @@ galaxy_info:
...
+      versions:
+        - archlinux
```
Comment (geerlingguy) [inline meta/main.yml:14]:
```suggestion
        - all
```

**Review by geerlingguy — CHANGES_REQUESTED** [review]

**Pull request merged**